### ISSUE TITLE
PSCommander - Program Files install fails to open config file

### ISSUE THREAD

**Issue by adamdriscoll** [ironmansoftware/powershell-universal#89]
**Describe the bug**

```
Install-Module PSCommander -Scope AllUsers
```

> I’ve installed PSCommander on my system but get the following when launching “Edit Config” from the icon in the tray:

> The term c:\program is not recognized as the name of a cmdlet, function, script file, or executable program …

**Version**

Tool (Visual Studio, VS Code, PSScriptPad, PS Module): PSCommander
Version Number: 1.0.2
Operating System: Windows

**Comment by rjgentile** [comment]
installing on another system does work when launching "Edit Conifg" from the icon; however ,changes made and saved to the config.ps1 file never appear to take effect. I've added blocks of code mentioned in the documentation (both "Tray Icon & Menu" and "Global HotKeys"), with no effect.

**Comment by adamdriscoll** [comment]
Can you do me a favor and exit PSCommander and then start it again (Start-Commander)? I'm just wondering if it's the file watcher that is failing or if it's the configuration that's failing.

**Comment by rjgentile** [comment]
it did not change anything ... wondering if our organization's move to OneDrive for our home folders is affecting it.
PSCommander folder is installed here ==> C:\Users\us13d3\OneDrive - Messer North America\Documents\PSCommander
PSCommander itself is installed here ==> C:\Users\us13d3\OneDrive - Messer North America\Documents\PowerShell\Modules\PSCommander\1.0.2

**Comment by adamdriscoll** [comment]
Certainly could be the case. We are using this to try and look up the folder path: 

```
$documents = [Environment]::GetFolderPath('MyDocuments')
$Folder = [IO.Path]::Combine($documents, "PSCommander")
$File = [IO.Path]::Combine($Folder, "config.ps1")
[IO.File]::Exists($File)
```

I think one thing that I'll get added is a way to specify an alternate path for this config file.

**Comment by rjgentile** [comment]
ok, I was able to change the config.ps1 file and the PSCommander right-click menu recognizes it! however, on this system, Edit Config still errors out. On the other system, it is ok .... I will keep investigating or maybe do a clean install on this system

**Comment by rjgentile** [comment]
nice tool, though ... like it

**Comment by adamdriscoll** [comment]
A fix for this has been released in 1.1.0
